app/code/Magento/Theme/Model/PageLayout/Config/Builder.php

```diff
@@ -27,6 +27,11 @@ class Builder implements \Magento\Framework\View\Model\PageLayout\Config\Builder
      */
     protected $themeCollection;
 
+    /**
+     * @var array
+     */
+    protected $configFiles = [];
```

Comment:
please change the access level to `private`

Reply:
For sure!